app/routes/select-your-business.js

```diff
@@ -46,7 +46,7 @@ module.exports = [{
       }
     },
     handler: async (request, h) => {
-      if (request.auth.credentials && (request.query.businessEmail !== request.auth.credentials.email)) {
```

Comment:
Should check both lowerCase to be safe

Reply:
No problem will add that in, but i initially left it out as `request.query.businessEmail` is set to lowercase by the Joi validation